internal/integrationtests/admin_api.go

```diff
@@ -26,14 +26,14 @@ type AdminApiIntegrationTestsInterface interface {
 }
 
 type CreateTenantRequest struct {
-	Name                string `json:"name"`
-	OwnerEmail          string `json:"owner_email"`
-	OwnerFirstName      string `json:"owner_first_name"`
-	OwnerLastName       string `json:"owner_last_name"`
-	OrganizationName    string `json:"organization_name"`
-	BaseURL             string `json:"base_url"`
-	SDPUIBaseURL        string `json:"sdp_ui_base_url"`
-	DistributionAccount string `json:"distribution_account"`
+	Name                    string `json:"name"`
+	OwnerEmail              string `json:"owner_email"`
+	OwnerFirstName          string `json:"owner_first_name"`
+	OwnerLastName           string `json:"owner_last_name"`
+	OrganizationName        string `json:"organization_name"`
+	DistributionAccountType string `json:"distribution_account_type"`
```

Comment:
💡 I think it's better to use schema.AccountType here rather than string, since it's the type you need this field to be when you use it.